packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -88,6 +88,13 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
     throw UnimplementedError('setPlaybackSpeed() has not been implemented.');
   }
 
+  /// Sets the max video resolution to a [resolution] value allowing the player
+  /// to reproduce lwoer resolution streams.
```

Comment:
nit: I think the wording on this could be improved. Something like 

```suggestion
  /// Sets the maximum allowed video resolution to [resolution], allowing the
  /// player to produce streams below this resolution.
```